src/OpenTK/opentk-1.0.projitems

```diff
@@ -0,0 +1,531 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+        <Compile Include="GlobalAssemblyInfo.cs"/>
```

Comment:
Why are these in a separate `.projitems` instead of in the `.csproj`?

A "feature" of Xamarin Studio is that it *won't* show sources from `<Import/>`ed projects. Meaning when using Xamarin Studio, you won't see ~any sources for this project.

For some things, like the mass of generated code in `src/Mono.Android`, that seems...okay? not great, but okay? Here that seems like a waste, so the `.projitems` file should probably be merged with the `.csproj`.